applications/ShapeOptimizationApplication/python_scripts/mesh_controller_with_solver.py

```diff
@@ -47,12 +52,14 @@ def __init__(self, MeshSolverSettings, model):
                     "verbosity" : 0,
                     "tolerance": 1e-7
                 },
-                "compute_reactions"         : false,
-                "calculate_mesh_velocity"   : false
+                "compute_reactions"                : false,
+                "calculate_mesh_velocity"          : false,
+                "reinitialize_model_part_each_step": false
             },
             "processes" : {
                 "boundary_conditions_process_list" : []
-            }
+            },
+            "automatic_mesh_refinement_settings": {}
```

Comment:
Its just naming but i think `automatic_remeshing_settings` fits better. The goal is not to make the mesh finer, but maintain the quality of the initial mesh after it got distorted by remeshing.

Reply:
done :)